pyproject.toml

```diff
@@ -24,8 +24,8 @@ classifiers = [
   "Operating System :: POSIX :: Linux",
 ]
 dependencies = [
-  "tmt~=1.35",
-  "fastapi~=0.112",
+  "tmt~=1.36",
+  "fastapi~=0.115",
```

Comment:
Not much worth locking dependencies, I think would be better to only `>=` if even necessary, github actions would pick up any incompatibilities, and probably there will be some packit packaging for the rest.

Reply:
hmm, probably not super important, but I'd keep it as is, unless you mean just specific packages. fwiw, they are not locked, well, only the major version is.